ecosystem/typescript/sdk/package.json

```diff
@@ -22,7 +22,7 @@
     "_build:browser": "tsup --platform browser --format iife --global-name aptosSDK --minify",
     "_build:node": "tsup --format cjs,esm --dts",
     "lint": "eslint \"**/*.ts\"",
-    "test": "jest",
+    "test": "ts-node ./scripts/publish_ans_contract.ts && jest",
```

Comment:
idk if we want to do this here right, then we publish every time even if you don't need to? Maybe this should be a separate command? Not super opinionated on this though.

Reply:
we do want to run this script every time we run tests - otherwise tests would fail.

Reply:
I mean if you make a local testnet, you only need to run this once right. But you can run the tests with `jest` multiple times after that. I imagine this is what you do when you're locally developing these tests?

Reply:
gotcha.